### PR TITLE
chore: Spelling

### PR DESCRIPTION
**Pull request by jsoref**
🎉 Thanks for submitting a pull request! 🎉

#### Summary

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/netlify-cli/commit/286819d03a48057df7f2fc4efe6bf908994eb1c8#commitcomment-87622073

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/netlify-cli/commit/d1431d91c3b6ffb94aac70616c889449fffc84b4
 
Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

For us to review and ship your PR efficiently, please perform the following steps:

- [ ] Open a [bug/issue](https://github.com/netlify/cli/issues/new/choose) before writing your code 🧑‍💻. This ensures we can discuss the changes and get feedback from everyone that should be involved. If you\`re fixing a typo or something that\`s on fire 🔥 (e.g. incident related), you can skip this step.
- [ ] Read the [contribution guidelines](../CONTRIBUTING.md) 📖. This ensures your code follows our style guide and
      passes our tests.
- [ ] Update or add tests (if any source code was changed or added) 🧪
- [ ] Update or add documentation (if features were changed or added) 📝
- [ ] Make sure the status checks below are successful ✅

**A picture of a cute animal (not mandatory, but encouraged)**

🦒

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -640,7 +640,7 @@ Generated by [`auto-changelog`](https://github.com/CookPete/auto-changelog).
 
 ### Features
 
-* support env write operations for sites opted in to beta env var experience ([#4751](https://github.com/netlify/cli/issues/4751)) ([e0695f9](https://github.com/netlify/cli/commit/e0695f94f0a54e6d31c627ea2b0b139895a21bc1))
+* support env write operations for sites opted into beta env var experience ([#4751](https://github.com/netlify/cli/issues/4751)) ([e0695f9](https://github.com/netlify/cli/commit/e0695f94f0a54e6d31c627ea2b0b139895a21bc1))
```
Comment (jsoref) [inline CHANGELOG.md:643]: I'm still working on `in to` / `into`. Happy to drop if these are objectionable.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7314,7 +7314,7 @@ netlify deploy --skip-functions-cache
 - Dev: Change functions timeout to 10s [`#725`](https://github.com/netlify/cli/pull/725)
 - Adds angular detector [`#721`](https://github.com/netlify/cli/pull/721)
 - fix: restore Netlify Functions logging [`#722`](https://github.com/netlify/cli/pull/722)
-- Tests: Dont asign NETLIFY_AUTH_TOKEN explicitly [`#718`](https://github.com/netlify/cli/pull/718)
+- Tests: Dont assign NETLIFY_AUTH_TOKEN explicitly [`#718`](https://github.com/netlify/cli/pull/718)
```
Comment (jsoref) [inline CHANGELOG.md:7317]: This seems correct

File: CHANGELOG.md [hunk 3]
```diff
@@ -7386,7 +7386,7 @@ netlify deploy --skip-functions-cache
 ### Commits
 
 - Netlify Dev: Handle error while killing child process [`5f61065`](https://github.com/netlify/cli/commit/5f61065e63cde864a5fa67d34065931bf4df9ef5)
-- Netilfy Dev: Pass down process env to child [`99934f5`](https://github.com/netlify/cli/commit/99934f56e2b1a1ea9aad04bb41d4668740b5dda8)
+- Netlify Dev: Pass down process env to child [`99934f5`](https://github.com/netlify/cli/commit/99934f56e2b1a1ea9aad04bb41d4668740b5dda8)
```
Comment (jsoref) [inline CHANGELOG.md:7389]: brand

File: CHANGELOG.md [hunk 4]
```diff
@@ -5588,7 +5588,7 @@ netlify deploy --skip-functions-cache
 
 ### Features
 
-* add functions directory creation when non-existent in netlify.toml ([#1996](https://www.github.com/netlify/cli/issues/1996)) ([4911e3d](https://www.github.com/netlify/cli/commit/4911e3d584af04f572150132cf694ae2049f3dc1))
+* add functions directory creation when nonexistent in netlify.toml ([#1996](https://www.github.com/netlify/cli/issues/1996)) ([4911e3d](https://www.github.com/netlify/cli/commit/4911e3d584af04f572150132cf694ae2049f3dc1))
```
Comment (jsoref) [inline CHANGELOG.md:5591]: Some projects don't like changing changelogs... happy to drop...

File: CHANGELOG.md [hunk 5]
```diff
@@ -7848,7 +7848,7 @@ netlify deploy --skip-functions-cache
 - Add global --json, --auth, && --silent flag support [`#371`](https://github.com/netlify/cli/pull/371)
 - [WIP] Add integration tests [`#254`](https://github.com/netlify/cli/pull/254)
 - Add postinstall success message [`#367`](https://github.com/netlify/cli/pull/367)
-- Remove Github token from browser history [`#364`](https://github.com/netlify/cli/pull/364)
+- Remove GitHub token from browser history [`#364`](https://github.com/netlify/cli/pull/364)
```
Comment (jsoref) [inline CHANGELOG.md:7851]: brand

**Review by jsoref — COMMENTED** [review]
Some corrections automatically suggested by Google Sheets. All fault mine

**Comment by danez** [comment]
I like the idea of your GitHub action, but please STOP spamming repositories with it. This PR was clearly automated and this is against the TOS (https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies#4-spam-and-inauthentic-activity-on-github) of GitHub.

**Comment by jsoref** [comment]
Each PR I make is done manually. I adjust the PR description to suit the parameters of each repository (otherwise it'd have a fixed formula, and this repository had a template). The choices I make for the corrections all involve me individually considering the corrections for each word or pattern. It's true that I'm using tooling to help, but I am not a robot, and there's a considerable amount of work involved on my part.

**Pull request closed without merging**